ripple

```diff
@@ -83,19 +83,40 @@ case ${1} in
         cp ./examples/manifest/extn-manifest-example.json ~/.ripple/firebolt-extn-manifest.json
 
         ## Update firebolt-extn-manifest.json
-        sed -i "" "s@\"default_path\": \"/usr/lib/rust/\"@\"default_path\": \"$workspace_dir/target/debug\"@" ~/.ripple/firebolt-extn-manifest.json
+        sed -i "" "s@\"default_path\": \"/usr/lib/rust/\"@\"default_path\": \"$workspace_dir/target/debug/\"@" ~/.ripple/firebolt-extn-manifest.json
         default_extension=$(get_default_extension)
         sed -i "" "s@\"default_extension\": \"so\"@\"default_extension\": \"$default_extension\"@" ~/.ripple/firebolt-extn-manifest.json
 
         ## Update firebolt-device-manifest.json
-        sed -i "" "s@\"library\": \"/etc/firebolt-app-library.json\"@\"library\": \"~/.ripple/firebolt-app-library.json\"@" ~/.ripple/firebolt-device-manifest.json
+        sed -i "" "s@\"library\": \"/etc/firebolt-app-library.json\"@\"library\": \"$HOME/.ripple/firebolt-app-library.json\"@" ~/.ripple/firebolt-device-manifest.json
 
         echo "All Done!"
     ;;
     "run")
         cargo build --features local_dev
         THUNDER_HOST=${2} cargo run --features local_dev core/main 
     ;;
+    "run-mock")
```

Comment:
Looks like some of options used in sed command works only on Mac. Need to make it compatible for Linux.

Reply:
probably a separate effort

Reply:
I was worried about the impact of using this on CI Pipeline. It's ok we we are not going to use this script on CI System.

Reply:
yes this is for developer runs